packages/flutter/lib/src/rendering/editable.dart

```diff
@@ -350,25 +350,26 @@ class RenderEditable extends RenderBox {
 
     if (_selection?.isValid == true) {
       if (_textPainter.getOffsetBefore(_selection.extentOffset) != null) {
-        config.addAction(SemanticsAction.moveCursorBackwardByCharacter, () {
-          final int offset = _textPainter.getOffsetBefore(_selection.extentOffset);
-          if (offset == null)
+        config.onMoveCursorBackwardByCharacter = (bool extentSelection) {
```

Comment:
I suspect it would be more performant to extract this closure into a private method

Reply:
Done.